### DISCUSSION TITLE
Error using http modelserver API v2

### DISCUSSION THREAD

**Discussion by julen9** [eclipse-emfcloud/emfcloud#223] [ModelServer]
Hi,

I am using the coffee-editor example.

I am trying to add new model files using http modelserver API v2, but, when I try to send a POST operation to my modelserver url, it send me this error:

![image](https://github.com/eclipse-emfcloud/emfcloud/assets/3693559/0188ae44-a1ed-43fa-b899-43f4ebaa35d2)

`Empty JSON`, but the body is not empty. Maybe I am not using the correct JSON format? Other operations like "undo", "redo", "Get models" works well, but not witt this POST operation.

please, could anyone help me?

Thanks in advance!
Best regards,
Julen.

**Comment by eneufeld** [comment]
Hi @julen9 , 

could you paste your post request?

Best,
Eugen
